cluster/router/condition/app_router_test.go

```diff
@@ -67,11 +67,11 @@ conditions:
 
 	_, err = z.Conn.Set(routerPath, []byte(testYML), 0)
 	assert.NoError(t, err)
-	defer func() {
-		err = ts.Stop()
-		assert.NoError(t, err)
-		z.Close()
-	}()
+	//defer func() {
```

Comment:
Detele them if useless

Reply:
yes, i will finish this when i have time